pkg/api/request.go

```diff
@@ -232,6 +232,8 @@ func (a *API) UserCreateRequest(w http.ResponseWriter, r *http.Request) {
 		err = apio.NewRequestError(err, http.StatusUnauthorized)
 	} else if err == accesssvc.ErrRuleNotFound {
 		err = apio.NewRequestError(fmt.Errorf("access rule %s not found", incomingRequest.AccessRuleId), http.StatusNotFound)
+	} else if err == accesssvc.ErrRequestOverlapsExistingGrant {
+		err = apio.NewRequestError(fmt.Errorf(err.Error()), http.StatusBadRequest)
```

Comment:
fmt.Errorf isn't needed here

```suggestion
		err = apio.NewRequestError(err.Error(), http.StatusBadRequest)
```